devito/arch/compiler.py

```diff
@@ -729,14 +730,60 @@ class IntelKNLCompiler(IntelCompiler):
     def __init__(self, *args, **kwargs):
         super(IntelKNLCompiler, self).__init__(*args, **kwargs)
 
-        self.cflags += ["-xMIC-AVX512"]
+        self.cflags.append('-xMIC-AVX512')
 
         language = kwargs.pop('language', configuration['language'])
 
         if language != 'openmp':
             warning("Running on Intel KNL without OpenMP is highly discouraged")
 
 
+class OneapiCompiler(IntelCompiler):
+
+    def __init__(self, *args, **kwargs):
+        super(OneapiCompiler, self).__init__(*args, **kwargs)
```

Comment:
Instead of 

```
super(OneapiCompiler, self).__init__(*args, **kwargs)
```

the following suffices:

```
super().__init__(*args, **kwargs)
```

Updating all instances of this pattern throughout the compiler, incrementally when I see them...

Reply:
ok